content/blog/2018/07/2018-07-10-jenkins-essentials-on-aws.adoc

```diff
@@ -0,0 +1,32 @@
+---
+layout: post
+title: Jenkins Essentials flavor for AWS
+author: batmat
+tags:
+- jenkinsessentials
+- essentials
```

Comment:
These two tags are the same tag, we should just use one consistently.

Reply:
I agree, but same I copied the practice from https://jenkins.io/blog/2018/06/26/jenkins-essentials-at-eclipsecon-france/